src/deltacode/__init__.py

```diff
@@ -312,3 +322,48 @@ def to_dict(self):
             ('new', new_file),
             ('old', old_file),
         ])
+
+class Stat(object):
+    """
+    Contains all the stats for the file changes in the new directory
+    with respect to the old directory.
+    """
+    def __init__(self, new_files_count, old_files_count):
+        self.new_files_count = new_files_count
+        self.old_files_count = old_files_count
+        self.added = 0
+        self.removed = 0
+        self.moved = 0
+        self.modified = 0
+        self.unmodified = 0
+        self.added_percent = 0
+        self.removed_percent = 0
+        self.moved_percent = 0
+        self.modified_percent = 0
+        self.unmodified_percent = 0
+
+    def calculate_stats(self):
+        """
+        Calculates the percentage change of new directory with respect to
+        the old directory.
+        """
+        self.added_percent = percent(self.added, self.old_files_count)
+        self.removed_percent = percent(self.removed, self.old_files_count)
+        self.moved_percent = percent(self.moved, self.old_files_count)
+        self.modified_percent = percent(self.modified, self.old_files_count)
+        self.unmodified_percent = percent(self.unmodified, self.old_files_count)
```

Comment:
Why don't we just do the math at this level instead of calling `percent(x, y)`

Reply:
@majurg since we need to calculate percent, again and again, I made a util function, such that we can use this in the future as well if we need it. Should I calculate here only and delete the percent function?

Reply:
@arnav-mandal1234 Yes, you are right. you should probably rename it `calculate_percent` or otherwise